### ISSUE TITLE
Unable to “fix match” merged seasons

### ISSUE THREAD

**Issue by Evil-Symbol** [JonnyWong16/plexpy#2010]
### Describe the Bug

I’ve been going through plex and fixing seasons that have been getting merged on tvdb - these are mostly anime that have been classified as 2 cour.

while fix match shows the correct episode to map to, it doesn’t actually seem to complete the change on submit.

### Steps to Reproduce

Existing show in plex with 2 seasons.
Update naming and merge into 1 (ie s02e01 now s01e13)
Refresh plex so it detects change
Open Tautulli, find the item in history
Click fix match
Select the correct Result and confirm.
Page refreshes and it looks like it works. but if you got back into history or F5, it’s still the only mapping 

### Expected Behavior

Map to episode as wizard presents.

### Tautulli Version

V2.11.1

### Git Branch

Master

### Git Commit Hash

V2.11.1

### Platform and Version

Docker

### Python Version

V3.10.10

### Browser and Version

Latest version chrome/safari 

### Link to Logs
`

2023-02-28 00:17:06 | WARNING | Unable to retrieve data for item_user_stats.
-- | -- | --
2023-02-28 00:17:01 | WARNING | Unable to retrieve data for item_user_stats.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53528 -> 53528.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53528 -> 53528 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53586 -> 53586.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53586 -> 53586 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53587 -> 53587.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53587 -> 53587 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53608 -> 53608.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53608 -> 53608 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53651 -> 53651.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53651 -> 53651 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53664 -> 53664.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53664 -> 53664 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53676 -> 53676.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53676 -> 53676 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53686 -> 53686.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53686 -> 53686 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53705 -> 53705.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53705 -> 53705 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53706 -> 53706.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53706 -> 53706 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53708 -> 53708.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53708 -> 53708 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Updating metadata in the database for rating_key 53732 -> 53732.
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53732 -> 53732 (episode)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53527 -> 53527 (season)
2023-02-28 00:16:59 | DEBUG | Tautulli DataFactory :: Mapping for rating_key 53526 -> 53526 (show)
2023-02-28 00:16:58 | INFO | Tautulli DataFactory :: Updating metadata in the database.

`

**Comment by samwiseg0** [comment]
Logs are always required. Please post your entire log.

**Comment by Evil-Symbol** [comment]
@samwiseg0 posted what I can via tablet. Not going to be near a PC for a few days now.

those messages are pretty consistent and was able to replicate on a brand new test install of both plex and Tautulli.

**Comment by xInsertx** [comment]
I noticed the same during my own quest to merge similar series, that had seasons squashed (mostly 2 cour type shows as well). I did fix a few mimicking the actions taken using sqlite editor - but its a lot of work - so would really appreciate it if this was resolved aswell.

One of the main reasons I use tautulli -  is for an offline copy of my watch history - which I have going back to 2015 (just when plexpy 1.0.1 was released).

While the other functionality is important - I've always been amazed of being able to look back (even prior to starting to also use trakt) - so being able to keep it neat and prevent kind of orphan history (which is kind of happening now for these shows) i feel would be a good thing.

that's ignoring any/all tvdb/tmdb naming nonsense. Just my 2cents.
